src/rsz/src/RepairDesign.cc

```diff
@@ -158,8 +161,21 @@ RepairDesign::repairDesign(double max_wire_length, // zero for none (meters)
   sta_->checkFanoutLimitPreamble();
 
   resizer_->incrementalParasiticsBegin();
+  int print_iteration = 0;
+  if (resizer_->level_drvr_vertices_.size() > 5 * max_print_interval_) {
+    print_interval_ = max_print_interval_;
+  } else {
+    print_interval_ = min_print_interval_;
+  }
+  if (verbose) {
+    printProgress(print_iteration, false, false, repaired_net_count);
+  }
   int max_length = resizer_->metersToDbu(max_wire_length);
   for (int i = resizer_->level_drvr_vertices_.size() - 1; i >= 0; i--) {
+    print_iteration++;
+    if (verbose) {
+      printProgress(print_iteration, false, false, repaired_net_count);
```

Comment:
How much output does this generate on a large design?

Reply:
I implemented some back off for larger designs to only print every 1000, but if it turns out to be too verbose we can look into different strategies for printing